include/eve/arch/riscv/rvv_utils.hpp

```diff
@@ -36,5 +36,8 @@ constexpr auto rvv_logical_ratio_v = []
 
 template<plain_scalar_value T>
 using rvv_m1_wide = wide<T, fixed<__riscv_v_fixed_vlen / 8 / sizeof(T)>>;
+
+// represents a wide that occupies the same size, but formed with int8_t.
+template<arithmetic_simd_value In> using rvv_i8_wide = wide<std::int8_t, fixed<sizeof(In)>>;
```

Comment:
Ate we sure about this ? Fixed<N> awaits a number of elements. Are tou wanting cardinal_t<In> instead ?

Reply:
Yep. I want to have a wide with the same size (that means the same lmul) that was given as template parameter. Here I use that `sizeof(std::int8_t)==1`, so `sizeof(In)` will be the number of `std::int8_t`, that I need to have the same size, as in input.

It is possible that some of these elements will be undefined. For example:
`wide<int, 1>` will occupy 1/2 physical vector register, so for vlen = 128 `rvv_i8_wide` will be `wide<std::int8_t, 8>`, and after `bit_cast` to new wide, I will have half of wide undefined.

This is needed as if I use `wide<std::int8_t, 4>`, that has that same `sizeof(scalar_type)*cardinal`, in intrinsics interface it will occupy 1/4 of physical register, so `bit_cast` will be impossible, but I need to use as underlying type `std::int8_t` (or `std::uint8_t`) as for this types full range of lmul values are available in intrinsics interface.

Reply:
Would you mind explaining that in detail in a comment cause, I know for sure, later on, someone will see that and think it is an error as it is rather peculiar.

Reply:
Added